examples/passport/login-with-nextjs/src/app/auth-event-handling/page.tsx

```diff
@@ -0,0 +1,243 @@
+    };
+    
+    fetchPassportProvider();
+  }, []);
+
+  // Set up accountsChanged event listener
+  useEffect(() => {
+    if (!provider) return;
+    
+    // Register event listener
+    provider.on(ProviderEvent.ACCOUNTS_CHANGED, handleAccountsChanged);
+    
+    // Log that event listener was registered
+    logEvent('provider_event_registered', { event: ProviderEvent.ACCOUNTS_CHANGED });
+    
+    // Cleanup function to remove event listener
+    return () => {
+      provider.removeListener(ProviderEvent.ACCOUNTS_CHANGED, handleAccountsChanged);
+    };
+  }, [provider, handleAccountsChanged, logEvent]);
+
+  // Handle login
+  const handleLogin = async () => {
+    try {
+      setLoading(true);
+      await passportInstance.login();
+      
+      // After login, get the provider
+      const provider = await passportInstance.connectEvm();
+      setProvider(provider);
+      
+      if (provider) {
+        // Get accounts
+        const accounts = await provider.request({ method: 'eth_requestAccounts' });
+        if (accounts && accounts.length > 0) {
+          setAddress(accounts[0]);
+          setAccountsState(accounts);
+          // Log the accounts change manually since the event might not fire
+          logEvent(ProviderEvent.ACCOUNTS_CHANGED, { accounts });
+        }
+        
+        // Get chain ID
+        const chainId = await provider.request({ method: 'eth_chainId' });
+        setChainId(chainId);
+      }
+      
+      setIsLoggedIn(true);
+    } catch (error) {
+      console.error('Login error:', error);
+      logEvent('login_error', error);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  // Handle logout
+  const handleLogout = async () => {
+    try {
+      setLoading(true);
+      await passportInstance.logout();
+      setIsLoggedIn(false);
+      setAddress('');
+      setChainId('');
+      setAccountsState([]);
+    } catch (error) {
+      console.error('Logout error:', error);
+      logEvent('logout_error', error);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  // Manually request accounts
+  const requestAccounts = async () => {
+    if (!provider) return;
+    
+    try {
+      setLoading(true);
+      const accounts = await provider.request({ 
+        method: 'eth_requestAccounts' 
+      });
+      logEvent('request_accounts', { accounts });
+      if (accounts && accounts.length > 0) {
+        setAddress(accounts[0]);
+        setAccountsState(accounts);
+      }
+    } catch (error) {
+      console.error('Request accounts error:', error);
+      logEvent('request_accounts_error', error);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  return (
+    <Stack gap="space.large">
+      <Heading>Passport SDK - Event Handling Example</Heading>
+      
+      <Stack gap="space.medium">
+        <Body>Status: {isLoggedIn ? 'Logged In' : 'Logged Out'}</Body>
+        
+        {isLoggedIn && address && (
+          <Stack gap="space.small">
```

Comment:
Can you use the Biom3 Table to display these values. Look at `login-with-etherjs` as a reference.